src/hooks/sections/useUpdateSection.ts

```
@@ -17,23 +21,35 @@ async function updateSection(req: UpdateSectionRequest) {
       break;
     case "BUNDLE":
     case "BUNK-JAMBO":
     case "NON-BUNK-JAMBO":
-      await updateSectionDoc(sessionId, sectionId, {
+      const sectionUpdates: UpdateData<SchedulingSectionDoc> = {
         name: updates.name,
         startDate: updates.startDate ? Timestamp.fromDate(updates.startDate.clone().startOf('day').toDate()) : undefined,
         endDate: updates.endDate ? Timestamp.fromDate(updates.endDate.clone().endOf('day').toDate()) : undefined,
         type: updates.type,
         publishedAt: null
+      }
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)